src/Controller/MenuController.java

```diff
@@ -335,4 +344,44 @@ public static void editRelationshipsControl(boolean shouldTerminate, final Diagr
             }
         }
     }
+    public static boolean autoCompleteOptionsControl(final String command, final Diagram diagram, boolean shouldTerminate) {
```

Comment:
Autocomplete options control is a misleading name, as it doesn't seem to be related to autocomplete as much as it is just executing the command

Reply:
True, I think I named it that initially before really thinking what it did but that's a good point.